CMakeLists.txt

```diff
@@ -1,37 +1,29 @@
 cmake_minimum_required(VERSION 3.15.0)
-project(libMATA)
-
-set(CMAKE_CXX_STANDARD 20)
+project(
+    libMATA
+    #VERSION ???? TODO: set the version automatically during merging
+    DESCRIPTION "A fast and simple automata library"
+    LANGUAGES C CXX)
```

Comment:
Fixed.

Reply:
Ok, not fixed, that `C` is needed there for cudd, otherwise the configuration fails.

Reply:
That is... interesting. Nothing to be done here, then. Maybe add a comment that the C here is solely for cudd and it can be removed when cudd is removed.

Reply:
Looks great. Thanks.